gpytorch/distributions/multivariate_normal.py

```diff
@@ -167,3 +190,31 @@ def __mul__(self, other):
 
     def __truediv__(self, other):
         return self.__mul__(1. / other)
+
+
+@register_kl(MultivariateNormal, MultivariateNormal)
```

Comment:
@gpleiss Does this mean that this will only work between two gpytorch MultivariateNormals? If so, in order to work with the pending variational refactor (specifically the variational distribution) we need to either:

1. Update this to do KL between a gpytorch MVN and any other MVN
2. Update gpytorch MultivariateNormal to accept `scale_tril` as a valid way of defining the covariance matrix.

Reply:
Actually, I guess making a gpytorch MVN with a `CholLazyTensor` will work at least as well. I'll merge this!